syntax = "proto2";
package column_type;

enum ColumnType {
    INT64 = 1;
    STRING = 2;
    FLOAT64 = 3;
}
